x-pack/plugins/security/public/session/session_timeout.tsx

```diff
@@ -41,7 +41,7 @@ export interface ISessionTimeout {
   extend(url: string): void;
 }
 
-export class SessionTimeout {
+export class SessionTimeout implements ISessionTimeout {
```

Comment:
🤦‍♂

Reply:
Haha, not a big deal though - I've heard platform will be abandoning `I{interface_name}` convention soon (with TS 3.8 upgrade) since we used it mostly to workaround the absence of proper "private fields" support and with TS 3.8 it'd not make too much sense to create a dedicated interface if it completely replicates public part of the class definition.